### PR TITLE
Georgia personal income tax rate schedule

### LINKED ISSUE #2251: Georgia personal income tax rate schedule
[Reference](https://dor.georgia.gov/tax-tables-georgia-tax-rate-schedule)

### PR DESCRIPTION
**Pull request by DianaShi5083**
Fixes #2251

Co-authored-by: PavelMakarchuk <PavelMakarchuk@users.noreply.github.com>

### Summary
📄💰🇬🇪

This pull request adds the YAML files for the Georgia state income tax rates for different filing statuses in the `policyengine_us/parameters/gov/states/ga/tax/income/main` directory. These files are needed to implement the state-level income tax for Georgia in the policy engine model.

> _`YAML` files added_
> _Georgia income tax rates_
> _Autumn leaves falling_

### Walkthrough
*  Add YAML files for Georgia income tax rate by filing status ([link](https://github.com/PolicyEngine/policyengine-us/pull/2252/files?diff=unified&w=0#diff-d94faf80967e453a53cad06c95caff7ca512f7412c2960dff2bc1b0ea18f025eL1-R33), [link](https://github.com/PolicyEngine/policyengine-us/pull/2252/files?diff=unified&w=0#diff-1f1308d6cce13b0a5e2d38bb3554c8a6cb375bbf602a008c1fb6c34a1fc426b5R1-R34), [link](https://github.com/PolicyEngine/policyengine-us/pull/2252/files?diff=unified&w=0#diff-f0ca38968155e70af21b10703ec68170bb52c15738ff8e2b523e69b38a4fd5d9L1-R33))
  - Define `joint.yaml` for married filing joint or head of household, with six brackets and a flat rate of 5.75% ([link](https://github.com/PolicyEngine/policyengine-us/pull/2252/files?diff=unified&w=0#diff-d94faf80967e453a53cad06c95caff7ca512f7412c2960dff2bc1b0ea18f025eL1-R33))
  - Define `separate.yaml` for married filing separate, with the same rate but different thresholds ([link](https://github.com/PolicyEngine/policyengine-us/pull/2252/files?diff=unified&w=0#diff-1f1308d6cce13b0a5e2d38bb3554c8a6cb375bbf602a008c1fb6c34a1fc426b5R1-R34))
  - Define `single.yaml` for single filers, with the same rate but different thresholds ([link](https://github.com/PolicyEngine/policyengine-us/pull/2252/files?diff=unified&w=0#diff-f0ca38968155e70af21b10703ec68170bb52c15738ff8e2b523e69b38a4fd5d9L1-R33))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+description: Georgia income tax rate for married filing joint or head of household.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml:1]:
```suggestion
description: Georgia provides the following tax rate for married couples filing jointly and for the head of the household.
```

File: policyengine_us/parameters/gov/states/ga/tax/income/main/separate.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+description: Georgia income tax rate for married filling separate.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/separate.yaml:1]:
```suggestion
description: Georgia provides the following tax rate for married couples filing separate.

```

File: policyengine_us/parameters/gov/states/ga/tax/income/main/single.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+description: Georgia income tax rate for single filers.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/single.yaml:1]:
```suggestion
description: Georgia provides the following tax rate for single filers.
```

File: policyengine_us/tests/policy/baseline/gov/states/ga/tax/income/ga_income_tax_before_refundable_credits.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- name: Single filer w/o income
+  period: 2022
+  input:
+    ga_taxable_income: 0
+    filing_status: SINGLE
+    state_code: GA
+  output:
+    ga_income_tax_before_refundable_credits: 0
+
+- name: JOINT filer
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/ga/tax/income/ga_income_tax_before_refundable_credits.yaml:19]:
```suggestion
- name: Joint filer 
```

File: policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+description: Georgia provides the following tax rate for married couples filing jointly and for the head of the household.
+ 
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Georgia income tax rate joint filers
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml:7]:
```suggestion
  label: Georgia income tax rate joint filers and head of household
```

File: policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml:1]: could you make this into two yaml files - one "joint.yaml" and one "head_of_household.yaml" -> same values

File: policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+description: Georgia provides the following tax rate for married couples filing jointly and for the head of the household.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/joint.yaml:1]:
```suggestion
description: Georgia taxes the personal income of joint filers at this rate.

File: policyengine_us/parameters/gov/states/ga/tax/income/main/separate.yaml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+description: Georgia provides the following tax rate for married couples filing separate.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/separate.yaml:1]:
```suggestion
description: Georgia taxes the personal income of married couples filing separate at this rate.

```

File: policyengine_us/parameters/gov/states/ga/tax/income/main/single.yaml [hunk 8]
```diff
@@ -0,0 +1,34 @@
+description: Georgia provides the following tax rate for single filers.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/single.yaml:1]:
```suggestion
description: Georgia taxes the personal income of single filers at this rate.
```

File: policyengine_us/variables/gov/states/ga/tax/income/ga_income_tax_before_refundable_credits.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ga/tax/income/ga_income_tax_before_refundable_credits.py:1]: add head of household case after parameter was added

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/variables/gov/states/ga/tax/income/ga_income_tax_before_refundable_credits.py
Comment (DianaShi5083) [reply policyengine_us/variables/gov/states/ga/tax/income/ga_income_tax_before_refundable_credits.py:1]: All done

File: policyengine_us/parameters/gov/states/ga/tax/income/main/head_of_household.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ga/tax/income/main/head_of_household.yaml:1]: add widow case

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**